rasa_nlu/persistor.py

```diff
@@ -47,11 +47,11 @@ def fetch_and_extract(self, filename):
 class AWSPersistor(Persistor):
     """Store models on S3 and fetch them when needed instead of storing them on the local disk."""
 
-    def __init__(self, data_dir, aws_region, bucket_name):
+    def __init__(self, data_dir, aws_region, bucket_name, endpoint_url):
         # type: (Text, Text, Text) -> None
         Persistor.__init__(self)
         self.data_dir = data_dir
-        self.s3 = boto3.resource('s3', region_name=aws_region)
+        self.s3 = boto3.resource('s3', region_name=aws_region, endpoint_url=endpoint_url)
```

Comment:
![image](https://user-images.githubusercontent.com/3448324/27794372-881be198-6034-11e7-83a3-f6a234d755b4.png)

Any AWS endpoint for me to test on? I think it should work based on the documents here